cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -23,7 +23,6 @@ import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.core.ProgramResult
 import com.github.ajalt.clikt.core.UsageError
 import com.github.ajalt.clikt.parameters.groups.mutuallyExclusiveOptions
-import com.github.ajalt.clikt.parameters.groups.required
```

Comment:
Note to myself: Looks like *detekt* regressed in detecting unused imports recently (https://github.com/detekt/detekt/issues/3020, https://github.com/detekt/detekt/issues/3074).